pkg/config/default.go

```diff
@@ -95,6 +95,10 @@ var (
 		parseSubnetPool("10.96.0.0/11", 24),
 		parseSubnetPool("10.128.0.0/9", 24),
 	}
+	// additionalHelperBinariesDir is an extra helper binaries directory that
+	// should be set during link-time, if different packagers put their
+	// helper binary in a different location
+	additionalHelperBinariesDir string
```

Comment:
Should this be a list rather then just one?

Reply:
I think the go linker only allows you to set strings? Not sure though. An alternative would be to take a string, split it by commas, and then put it into a list to allow setting more than one additional helper binary dir.

Reply:
Ok, I thought this was a function call.
LGTM